x/gateway/types/genesis.go

```diff
@@ -22,29 +22,29 @@ func DefaultGenesis() *GenesisState {
 func (gs GenesisState) Validate() error {
 	gatewayIndexMap := make(map[string]struct{})
 
-	for _, elem := range gs.GatewayList {
+	for _, gateway := range gs.GatewayList {
```

Comment:
See https://github.com/pokt-network/poktroll/pull/72/files#r1364288032 think better to leave this to avoid conflicts